docs/zh_cn/advanced_guides/datasets/kitti_det.md

```diff
@@ -78,39 +76,42 @@ kitti
 ├── kitti_dbinfos_train.pkl
 ├── kitti_infos_test.pkl
 ├── kitti_infos_trainval.pkl
-├── kitti_infos_train_mono3d.coco.json
-├── kitti_infos_trainval_mono3d.coco.json
-├── kitti_infos_test_mono3d.coco.json
-├── kitti_infos_val_mono3d.coco.json
 ```
 
 其中的各项文件的含义如下所示：
 
-- `kitti_gt_database/xxxxx.bin`： 训练数据集中包含在 3D 标注框中的点云数据
-- `kitti_infos_train.pkl`：训练数据集的信息，其中每一帧的信息包含下面的内容：
-  - info\['point_cloud'\]: {'num_features': 4, 'velodyne_path': velodyne_path}.
-  - info\['annos'\]: {
-    - 位置：其中 x,y,z 为相机参考坐标系下的目标的底部中心（单位为米），是一个尺寸为 Nx3 的数组
-    - 维度: 目标的高、宽、长（单位为米），是一个尺寸为 Nx3 的数组
-    - 旋转角：相机坐标系下目标绕着 Y 轴的旋转角 ry，其取值范围为 \[-pi..pi\] ，是一个尺寸为 N 的数组
-    - 名称：标准框所包含的目标的名称，是一个尺寸为 N 的数组
-    - 困难度：kitti 官方所定义的困难度，包括 简单，适中，困难
-    - 组别标识符：用于多部件的目标
-      }
-  - (optional) info\['calib'\]: {
-    - P0：校对后的 camera0 投影矩阵，是一个 3x4 数组
-    - P1：校对后的 camera1 投影矩阵，是一个 3x4 数组
-    - P2：校对后的 camera2 投影矩阵，是一个 3x4 数组
-    - P3：校对后的 camera3 投影矩阵，是一个 3x4 数组
-    - R0_rect：校准旋转矩阵，是一个 4x4 数组
-    - Tr_velo_to_cam：从 Velodyne 坐标到相机坐标的变换矩阵，是一个 4x4 数组
-    - Tr_imu_to_velo：从 IMU 坐标到 Velodyne 坐标的变换矩阵，是一个 4x4 数组
-      }
-  - (optional) info\['image'\]:{'image_idx': idx, 'image_path': image_path, 'image_shape', image_shape}.
-
-**注意**：其中的 info\['annos'\] 中的数据均位于相机参考坐标系中，更多的细节请参考[此处](http://www.cvlibs.net/publications/Geiger2013IJRR.pdf)。
-
-获取 kitti_infos_xxx.pkl 和 kitti_infos_xxx_mono3d.coco.json 的核心函数分别为 [get_kitti_image_info](https://github.com/open-mmlab/mmdetection3d/blob/7873c8f62b99314f35079f369d1dab8d63f8a3ce/tools/data_converter/kitti_data_utils.py#L140) 和 [get_2d_boxes](https://github.com/open-mmlab/mmdetection3d/blob/7873c8f62b99314f35079f369d1dab8d63f8a3ce/tools/data_converter/kitti_converter.py#L378).
+- `kitti_gt_database/xxxxx.bin`：训练数据集中包含在 3D 标注框中的点云数据
+- `kitti_infos_train.pkl`：训练数据集，其中每一帧的信息包含 `metainfo` 和 `data_list` 两个键值。`metainfo` 是一个字典，它包含数据集的基本信息，例如 `CLASSES` 和 `version`。`data_list` 是一个列表，它包含了所有需要的数据信息，每一个元素是一个样本的详细信息字典。详细信息如下所示：
```

Comment:
We should update English doc here to:
 training dataset, a dict contains two keys: `metainfo` and `data_list`.
   `metainfo` contains the basic information for the dataset itself, such as `DATASET`,`CLASSES` and `version`, while `data_list` is a list of dict, each dict ( hereinafter referred to as`info`) contains all the detailed information of single sample as follows:

And update Chinese documentation accordingly